packages/builders/src/messages/embed/UnsafeEmbed.ts

```diff
@@ -105,7 +148,7 @@ export class UnsafeEmbed implements APIEmbed {
 		return (
 			(this.title?.length ?? 0) +
 			(this.description?.length ?? 0) +
-			this.fields.reduce((prev, curr) => prev + curr.name.length + curr.value.length, 0) +
+			(this.fields ?? []).reduce((prev, curr) => prev + curr.name.length + curr.value.length, 0) +
 			(this.footer?.text.length ?? 0) +
 			(this.author?.name.length ?? 0)
 		);
```

Comment:
Shouldn't all of these be using the `data` property instead of the getters?
```suggestion
		return (
			(this.data.title?.length ?? 0) +
			(this.data.description?.length ?? 0) +
			(this.data.fields ?? []).reduce((prev, curr) => prev + curr.name.length + curr.value.length, 0) +
			(this.data.footer?.text.length ?? 0) +
			(this.data.author?.name.length ?? 0)
		);
```